include/lsst/meas/base/PixelFlags.h

```diff
@@ -42,13 +41,20 @@ namespace lsst { namespace meas { namespace base {
  */
 class PixelFlagsControl {
 public:
-
+    LSST_CONTROL_FIELD(center, std::vector<std::string>,
+                       "List of mask planes for which to search center of footprint");
+    LSST_CONTROL_FIELD(any, std::vector<std::string>,
+                       "List of mask planes for which to search entire footprint");
     /**
      *  @brief Default constructor
      *
      *  All control classes should define a default constructor that sets all fields to their default values.
```

Comment:
So why aren't the default values for the two fields defined above being set here?